src/main/scala/org/apache/spark/sql/execution/datasources/greenplum/GreenplumUtils.scala

```diff
@@ -66,8 +67,13 @@ object GreenplumUtils extends Logging {
    * @param df the [[DataFrame]] will be copy to the Greenplum
    * @param schema the table schema in Greemnplum
    * @param options Options for the Greenplum data source
+   * @param isAppend whether append data to this gptable
    */
-  def copyToGreenplum(df: DataFrame, schema: StructType, options: GreenplumOptions): Unit = {
+  def copyToGreenplum(
+      df: DataFrame,
+      schema: StructType,
+      options: GreenplumOptions,
+      isAppend: Boolean): Unit = {
```

Comment:
can  we just def two method appendToGP and overwrite to gp to avoid so many `if(isAppend)`